### ISSUE TITLE
Python 3.2 method renaming / changes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [testing-cabal/unittest-ext#55]
```
See http://bugs.python.org/issue9424
```

Original issue reported on code.google.com by `fuzzyman` on 3 Jan 2011 at 6:06

**Comment by GoogleCodeExporter** [comment]
```
Oops - incorrect Python issue. The correct one is:
http://bugs.python.org/issue10273
```

Original comment by `fuzzyman` on 3 Jan 2011 at 6:07

**Comment by GoogleCodeExporter** [comment]
Original comment by `ezio.melotti` on 22 Feb 2011 at 1:23

**Comment by GoogleCodeExporter** [comment]
```
I renamed assert[Not]RegexpMatches to assert[Not]Regex and 
assert[Raises|Warns]Regexp to assert[Raises|Warns]Regex in 
http://hg.python.org/unittest2/rev/27d6829cd8fa and added 
PendingDeprecationWarnings to the old names.
assertWarnsRegexp is not yet released (in unittest2-0.5.1) so it didn't need 
the deprecation.
```

Original comment by `ezio.melotti` on 16 Mar 2011 at 4:56
- Changed state: **Fixed**
